docs/community/bindings.md

```diff
@@ -2,6 +2,10 @@
 
 As a C library, bindings can be made to call H3 functions from different programming languages. This page lists different bindings currently available. Contributions to this list are welcome, please feel free to open a [pull request](https://github.com/uber/h3/tree/master/docs/community/bindings.md).
 
+## C-sharp 
```

Comment:
```suggestion
## C\#
```

This would probably produce the best result? Most markdown parsers understand backslash escaping.

Reply:
I tried that and it seemed to not be working, I didn't look into why.